man/avocado.rst

```diff
@@ -104,83 +136,91 @@ Options for subcommand `run` (`avocado run --help`)::
       --open-browser        Open the generated report on your preferred browser.
                             This works even if --html was not explicitly passed,
                             since an HTML report is always generated on the job
-                            results dir. Current: False
-      --html-job-result {on,off}
+                            results dir.
+      --disable-html-job-result
                             Enables default HTML result in the job results
-                            directory. File will be located at
-                            "html/results.html".
+                            directory. File will be named "results.html".
       --journal             Records test status changes (for use with avocado-
                             journal-replay and avocado-server)
       --json FILE           Enable JSON result format and write it to FILE. Use
-                            '-' to redirect to the standard output.
-      --json-job-result {on,off}
+                            "-" to redirect to the standard output.
+      --disable-json-job-result
                             Enables default JSON result in the job results
                             directory. File will be named "results.json".
-      --tap FILE            Enable TAP result output and write it to FILE. Use '-'
-                            to redirect to the standard output.
-      --tap-job-result {on,off}
+      --tap FILE            Enable TAP result output and write it to FILE. Use "-"
+                            to redirect to standard output.
+      --disable-tap-job-result
                             Enables default TAP result in the job results
-                            directory. File will be named "results.tap".
-      --tap-include-logs    Include test logs as comments in TAP output. Defaults
-                            to False
+                            directory. File will be named "results.tap"
+      --tap-include-logs    Include test logs as comments in TAP output.
+      -z, --archive         Archive (ZIP) files generated by tests
+
+    output and result format:
       --xunit FILE          Enable xUnit result format and write it to FILE. Use
-                            '-' to redirect to the standard output.
-      --xunit-job-result {on,off}
+                            "-" to redirect to the standard output.
+      --disable-xunit-job-result
                             Enables default xUnit result in the job results
                             directory. File will be named "results.xml".
-      --xunit-job-name XUNIT_JOB_NAME
+      --xunit-job-name JOB.RUN.RESULT.XUNIT.JOB_NAME
```

Comment:
I understand some of these changes were either automated or (rightfully so) intended to sync with the output `--help` produces, but this specific change makes the command line option harder to read.

So, I decided to address that (initially for `avocado run --help`) [here](https://github.com/avocado-framework/avocado/pull/4587).

Maybe if/when it gets merged, a refresh here would automatically solve this "regression" of sorts.

Reply:
Perfect, it's been merged already, I'm going to update the PR with that.